oxenss/rpc/oxend_rpc.cpp

```diff
@@ -51,8 +51,10 @@ oxend_seckeys get_sn_privkeys(
                                                 (data.empty() ? "no data received" : data[0])};
                                     }
                                     auto r = nlohmann::json::parse(data[1]);
+                                    auto pk_it = r.find("service_node_privkey");
+                                    const std::string& pk =
+                                            pk_it == r.end() ? "" : pk_it->get_ref<std::string&>();
```

Comment:
Does this end up being a reference to a temporary `std::string` when `pk_it == r.end()`?  I'm not sure that it is, but I'm also not sure that it *isn't*.

nlohmann::json has a get-with-default method, `value`, that would resolve that and simplify it a bit with a switch to string_views instead of strings:
```suggestion
                                    auto pk = r.value<std::string_view>("service_node_privkey", "");
```

Reply:
(I've pushed this change to the PR).